### ISSUE TITLE
hlx up fails on nodegit using CI build of hlx

### ISSUE THREAD

**Issue by cwoeltge** [adobe/helix-cli#207]
after installing CI build of `hlx` with the installer script, `hlx up` fails:

```  
  $ hlx up
  Cannot find module 'nodegit'
```

`npm install nodegit` complains about vulnerabilities

**Comment by cwoeltge** [comment]
after installing 0.7.7 with the installer script, hex up fails:
  
  $ hlx up
  Cannot find module 'nodegit'

`npm install nodegit` complains about vulnerabilities

**Comment by cwoeltge** [comment]
```$ npm install nodegit
npm WARN deprecated hoek@2.16.3: The major version is no longer supported. Please update to 4.x or newer

> nodegit@0.22.2 install /Users/mahatmax/development/adobe/helix/helix-pt-1/node_modules/nodegit
> node lifecycleScripts/preinstall && node lifecycleScripts/install

[nodegit] Running pre-install script
[nodegit] Configuring libssh2.
[nodegit] Running install script
[nodegit] Success: "/Users/mahatmax/development/adobe/helix/helix-pt-1/node_modules/nodegit/build/Release/nodegit.node" is installed via remote
[nodegit] Completed installation successfully.

> nodegit@0.22.2 postinstall /Users/mahatmax/development/adobe/helix/helix-pt-1/node_modules/nodegit
> node lifecycleScripts/postinstall

+ nodegit@0.22.2
updated 1 package and audited 12080 packages in 31.666s
found 8 moderate severity vulnerabilities
  run `npm audit fix` to fix them, or `npm audit` for details
$ hlx up
Cannot find module 'nodegit'
```

**Comment by tripodsan** [comment]
@cwoeltge can you try again with the properly released https://github.com/adobe/helix-cli/releases/tag/v0.7.7?

**Comment by stefan-guggisberg** [comment]
> npm WARN deprecated hoek@2.16.3: The major version is no longer supported. Please update to 4.x or newer

@cwoeltge that's known and has been fixed in #204.

**Comment by stefan-guggisberg** [comment]
created a more specific issue #208
